<?xml version="1.0" encoding="UTF-8"?><testrun name="CashcardJsonTest" project="cashcardprueba" tests="1" started="1" failures="1" errors="0" ignored="0">
  <testsuite name="com.cashcard.cashcardprueba.CashcardJsonTest" time="0.147" displayname="CashcardJsonTest" uniqueid="[engine:junit-jupiter]/[class:com.cashcard.cashcardprueba.CashcardJsonTest]">
    <testcase name="myFirstTest" classname="com.cashcard.cashcardprueba.CashcardJsonTest" time="0.147" displayname="myFirstTest()" uniqueid="[engine:junit-jupiter]/[class:com.cashcard.cashcardprueba.CashcardJsonTest]/[method:myFirstTest()]">
      <failure>
        <expected>42</expected>
        <actual>1</actual>
        org.opentest4j.AssertionFailedError: 
expected: 42
 but was: 1
	at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.base/java.lang.reflect.Constructor.newInstanceWithCaller(Constructor.java:499)
	at com.cashcard.cashcardprueba.CashcardJsonTest.myFirstTest(CashcardpruebaApplicationTests.java:11)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)


      </failure>
    </testcase>
  </testsuite>
</testrun>
